server/neptune/workflows/internal/terraform/state/workflow.go

```diff
@@ -89,6 +89,8 @@ type WorkflowResult struct {
 }
 
 type Workflow struct {
+	// Mode is used to determine which jobs are populated
```

Comment:
i mean this is a pretty barebones comment lmao, i meant something a bit more in depth, like "plan and validate jobs are only populated when it's PR mode, etc."

Reply:
@nishkrishnan added some detail